sqldelight-gradle-plugin/src/test/integration/src/androidTest/java/com/squareup/sqldelight/integration/IntegrationTests.java

```diff
@@ -1,147 +1,110 @@
+    assertThat(person).isEqualTo(new AutoValue_MyPerson(4, "Bob", "Bob"));
   }
 
   @Test public void indexedArgLastTwo() {
     // First arg declared is ?, second arg declared is ?2.
-    SqlDelightStatement indexedArgLast = Person.FACTORY.indexed_arg_last_2("Alec", "Strong");
-    Cursor cursor = database.rawQuery(indexedArgLast.statement, indexedArgLast.args);
-    assertThat(cursor.getCount()).isEqualTo(1);
-    cursor.moveToFirst();
-    Person person = Person.FACTORY.equivalent_namesMapper().map(cursor);
-    assertThat(person).isEqualTo(new AutoValue_Person(1, "Alec", "Strong"));
+    Person person = personQueries.indexedArgLast2("Alec", "Strong", AutoValue_MyPerson::new).executeAsOne();
+    assertThat(person).isEqualTo(new AutoValue_MyPerson(1, "Alec", "Strong"));
   }
 
   @Test public void nameIn() {
-    SqlDelightStatement nameIn = Person.FACTORY.name_in(new String[] { "Alec", "Matt", "Jake" });
-    Cursor cursor = database.rawQuery(nameIn.statement, nameIn.args);
-    assertThat(cursor.getCount()).isEqualTo(3);
+    List<Person> people = personQueries.nameIn(Arrays.asList("Alec", "Matt", "Jake")).executeAsList();
+    assertThat(people).hasSize(3);
   }
 
   @Test public void sqliteKeywordQuery() {
-    SqlDelightStatement selectAll = SqliteKeywords.FACTORY.select_all();
-    Cursor cursor = database.rawQuery(selectAll.statement, selectAll.args);
-    assertThat(cursor.getCount()).isEqualTo(1);
-    cursor.moveToFirst();
-    SqliteKeywords sqliteKeywords = SqliteKeywords.FACTORY.select_allMapper().map(cursor);
-    assertThat(sqliteKeywords).isEqualTo(new AutoValue_SqliteKeywords(1, 10, 20));
-  }
-
-  @Test public void sqliteKeywordColumnString() {
-    SqlDelightStatement selectAll = SqliteKeywords.FACTORY.select_all();
-    Cursor cursor = database.rawQuery(selectAll.statement, selectAll.args);
-    assertThat(cursor.getCount()).isEqualTo(1);
-    cursor.moveToFirst();
-    long where = cursor.getLong(cursor.getColumnIndexOrThrow(SqliteKeywords.WHERE));
-    assertThat(where).isEqualTo(10);
+    SqliteKeywords keywords = keywordsQueries.selectAll(AutoValue_SqliteKeywords::new).executeAsOne();
+    assertThat(keywords).isEqualTo(new AutoValue_SqliteKeywords(1, 10, 20));
   }
 
   @Test public void compiledStatement() {
-    SqliteKeywords.Insert_stmt statement = new SqliteKeywords.Insert_stmt(database);
-    statement.bind(11, 21);
-    statement.program.executeInsert();
-    statement.bind(12, 22);
-    statement.program.executeInsert();
-
-    SqlDelightStatement selectAll = SqliteKeywords.FACTORY.select_all();
-    Cursor cursor = database.rawQuery(selectAll.statement, selectAll.args);
+    keywordsQueries.insertStmt(11, 21);
+    keywordsQueries.insertStmt(12, 22);
+
     long current = 10;
-    while (cursor.moveToNext()) {
-      assertThat(cursor.getLong(cursor.getColumnIndexOrThrow(SqliteKeywords.WHERE))).isEqualTo(current++);
+    for (_group_ group : keywordsQueries.selectAll().executeAsList()) {
```

Comment:
👀

Reply:
if people want to name their tables `\`group\`` they deserve this

Reply:
how do i escape ` inside markdown

Reply:
I see you figured it out. Let's see if i can figure out how to show you how to escape markdown inside markdown...

``` `` ` `` ```

Reply:
Ye

Reply:
nice how did you do that

Reply:
```` ``` `` `group` `` ``` ````